plugins/metadata/mods34/locale/fr_CA/locale.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/fr_CA/locale.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
```

Comment:
Copyright dates should end in 2018 (both this one and below), and should be `Simon Fraser University` rather than `Simon Fraser University Library`.